app/pages/sections/additionalInformation/AdditionalInformationSection.scala

```diff
@@ -17,10 +17,10 @@
 package pages.sections.additionalInformation
 
 import models.Index
-import pages.sections.Section
+import pages.QuestionPage
 import play.api.libs.json.{JsObject, JsPath}
 
-case class AdditionalInformationSection(informationIndex: Index) extends Section[JsObject] {
+case class AdditionalInformationSection(informationIndex: Index) extends QuestionPage[JsObject] {
```

Comment:
Sorry, my bad, this should be a `Section[JsObject]`

Reply:
Yep, reverted back now